tasks/src/app/simple-button/simple-button.component.ts

```diff
@@ -3,20 +3,30 @@ import { Component, OnInit } from '@angular/core';
 @Component({
 	selector: 'simple-button',
 	templateUrl: './simple-button.component.html',
-	styleUrls: ['./simple-button.component.css']
+	styleUrls: ['./simple-button.component.css'],
 })
 export class SimpleButtonComponent implements OnInit {
-	public numberOfClicks: number;
+	public isInputEnable: boolean;
+	public greetingText: string;
 
 	public constructor() {
-		this.numberOfClicks = 0;
+		this.isInputEnable = false;
+		this.greetingText = "";
 	}
 
 	public ngOnInit(): void {
 	}
 
-	public increseNumberOfCounts(): void {
-		this.numberOfClicks++;
+	public changeInputStatus(): void {
+		this.isInputEnable = !this.isInputEnable;
+		if (this.isInputEnable === false) {
```

Comment:
```suggestion
		if (!this.isInputEnable) {
```
you can verify in this way